pkg/util/image_prep_utils.go

```diff
@@ -79,29 +75,17 @@ func getImageFromTar(tarPath string) (string, error) {
 func CleanupImage(image Image) {
 	if image.FSPath != "" {
 		glog.Infof("Removing image filesystem directory %s from system", image.FSPath)
-		errMsg := remove(image.FSPath, true)
-		if errMsg != "" {
-			glog.Error(errMsg)
+		if err := remove(image.FSPath, true); err != nil {
+			glog.Error(err.Error())
 		}
 	}
 }
 
-func remove(path string, dir bool) string {
-	var errStr string
-	if path == "" {
-		return ""
-	}
-
-	var err error
+func remove(path string, dir bool) error {
```

Comment:
Remove remove.  os.RemoveAll tries to call remove before it deletes the directory.  Just calling that directly will be fine

https://golang.org/src/os/path.go?s=1634:1667#L56

Reply:
good call, done